paddle/fluid/framework/ir/mkldnn/compute_propagate_scales_mkldnn_pass.cc

```diff
@@ -336,27 +336,46 @@ void ComputePropagateScalesMkldnnPass::ComputeWeightScales(
   ComputeLstmWeightScales(graph, scope, "WeightX", "WeightH", var_quant_scales);
 }
 
-void ComputePropagateScalesMkldnnPass::UpdateScaleOpInScale(
+void ComputePropagateScalesMkldnnPass::UpdateScaleOpInOutScales(
```

Comment:
I think that for a new quantization method where there is linear_quantize before and and linear_dequantize after each operator this UpdateScaleOp it is no longer needed so much.